packages/cli/internal/pkg/cli/workflow_run.go

```diff
@@ -75,8 +82,16 @@ using input parameters contained in file "file:///Users/ec2-user/myproj/test-arg
 		}),
 		ValidArgsFunction: NewWorkflowAutoComplete().GetWorkflowAutoComplete(),
 	}
-	cmd.Flags().StringVarP(&vars.Arguments, argsFlag, argsFlagShort, "", argsFlagDescription)
+	cmd.Flags().StringVarP(&vars.Arguments, inputsFileFlag, inputsFileFlagShort, "", inputsFileFlagDescription)
 	cmd.Flags().StringVarP(&vars.ContextName, contextFlag, contextFlagShort, "", contextFlagDescription)
+	aliasFn := func(f *pflag.FlagSet, name string) pflag.NormalizedName {
```

Comment:
Do we want to deprecate existing flag?

https://github.com/spf13/pflag#deprecating-a-flag-or-its-shorthand

Normalization doesn't look like a good fit for this. Also, I think we may want to keep original behavior of --args for backward compatibility.

Reply:
We want to alias it instead of explicitly deprecating it. This is the official doc for the use case (Example 2): https://pkg.go.dev/github.com/spf13/pflag#readme-mutating-or-normalizing-flag-names